browser/app/js/components/Browse.js

```diff
@@ -480,338 +486,309 @@ export default class Browse extends React.Component {
+
+                  <ObjectsList dataType={ this.dataType.bind(this) }
+                               selectPrefix={ this.selectPrefix.bind(this) }
+                               showDeleteConfirmation={ this.showDeleteConfirmation.bind(this) }
+                               shareObject={ this.shareObject.bind(this) }
+                               checkObject={ this.checkObject.bind(this) }
+                               checkedObjectsArray={ checkedObjects }
+                               showObjectPreview={ this.showPreview.bind(this) }
+                  />
+                </InfiniteScroll>
+
+                <div className="text-center" style={ { display: (istruncated && currentBucket) ? 'block' : 'none' } }>
+                  <span>Loading...</span>
                 </div>
-                <div className="fesl-item fesl-item-actions"></div>
-              </header>
+              </Dropzone>
             </div>
-            <div className="feb-container">
-              <InfiniteScroll loadMore={ this.listObjects.bind(this) }
-                hasMore={ istruncated }
-                useWindow={ true }
-                initialLoad={ false }>
-                <ObjectsList dataType={ this.dataType.bind(this) }
-                  selectPrefix={ this.selectPrefix.bind(this) }
-                  showDeleteConfirmation={ this.showDeleteConfirmation.bind(this) }
-                  shareObject={ this.shareObject.bind(this) }
-                  checkObject={ this.checkObject.bind(this) }
-                  checkedObjectsArray={ checkedObjects } />
-              </InfiniteScroll>
-              <div className="text-center" style={ { display: (istruncated && currentBucket) ? 'block' : 'none' } }>
-                <span>Loading...</span>
+          </div>
+
+          <aside className={ classNames ({ 'preview': true, 'preview--toggled': previewStatus }) }>
+            <div className="preview__header">
+              <span className="preview__label">filename.jpg</span>
+              <i className="preview__close zmdi zmdi-long-arrow-left" onClick={this.hidePreview.bind(this)}/>
+            </div>
+
+            <div className="preview__body">
+              <div className="preview__item preview__item--img">
+                <img src="https://placeholdit.imgix.net/~text?w=100&h=300" alt=""/>
+              </div>
+
+              <div className="preview__info">
+                <dl>
+                  <dt>Type</dt>
+                  <dd>JPEG Image</dd>
+
+                  <dt>Size</dt>
+                  <dd>1.1 MB</dd>
+
+                  <dt>Modified</dt>
+                  <dd>Mar 1, 2017 12:!2 PM</dd>
+
+                  <dt>Policy</dt>
+                  <dd>Read and Write</dd>
+
+                  <dt>Shared</dt>
+                  <dd>No</dd>
+                </dl>
+              </div>
+
+              <div className="preview__actions">
+                <button className="btn btn--lg btn--primary">Preview</button>
```

Comment:
What's the intent of this button if the preview is shown above? It's like having "Download" when the object is already downloading, no?

![screen shot 2017-07-19 at 4 05 43 pm](https://user-images.githubusercontent.com/3038254/28393547-2683f492-6c9c-11e7-89d5-228f928874f6.png)